content/articles/likelihood-function-in-hospital-dynamics-using- r/index.md

```diff
@@ -0,0 +1,102 @@
+
+The probability level is expressed by the likelihood function, which is a mathematical function. Suppose we take a set of observations as fixed; they have already happened and now we figure out which set of model parameters we are most likely to observe them under.
+
+To produce the data, we employ an Equal Chance of Having an Outcome, however, we'll suppose we don't know this for now. For practically all diagnostic scenarios, we're forced to estimate the processes that produce the data we're looking at, which is why we're required to estimate these parameters..
+
+###  Understanding the linear and Poisson distribution
+
+A regression model that employs a straight line to explain the relationship between variables is known as linear regression. It searches for the value of the regression coefficient(s) which minimizes the total error of the model to find the line of best fit through your data.Likelihood of a particular number of events occurring in a given time frame. Consider the number of individuals in a population who are in declining health. 
+
+The Poisson distribution expresses the likelihood of poor health individuals amidst those who are in good health.
+
+### Maximizing the likelihood
+MLE is a method used to estimate characteristics of the target population (such as mean and variance for Normal, rate (lambda) for Poisson, and so forth) from sample data with the objective of maximizing the chance (likelihood) of obtaining the observed values.
+
+Maximizing a function is just the same as using the log of that function multiplied by a negative one.
+
+![Maxiximazing the likelihood](/engineering-education/likelihood-function-in-hospital-dynamics-using-r/max.png)
+
+### Finding out model coefficient using MLE
+
+The regression correlation coefficient can tell you how closely your slope is to the correlation value, and it's a useful metric for regressive target prediction. It specifies how far the variables deviate from the assumed distribution.
+
+Observe how one might manage the budgeted funds and improve how to treat each sickness judiciously in a hospital setting.
+
+![model coefficient using MLE](/engineering-education/likelihood-function-in-hospital-dynamics-using-r/.png)
+
+### Standard linear model  and Poisson model
+When studying the link between interval scale predictors and interval scale results, statisticians typically use linear regression.
+
+y=mx+c, x, and y are the two variables, and m and c, are the two coefficients. The gradient of the line is denoted by the coefficient m, and the y-intercept is represented by the coefficient c. The value of y obtained at x=0 is referred to as the intercept. Similarly, if you increase the x-value by one unit, the y-value increases by m units; if you increase the x-value by one unit, the y-value increases by m units; if you increase the x-value by one unit, the y-value increases by m units; if you increase the x-value by one unit, the y-value increases by m units.

```

Comment:
Fix any grammatical errors. Also, split the sentence into two sections to make it easier for the reader to understand.